classes/label.rb

```diff
@@ -0,0 +1,17 @@
+require_relative 'item'
+
+class Label
+  attr_reader :title, :color, :items, :id
+
+  def initialize(title, color)
+    @id = rand(1000)
+    @title = title
+    @color = color
+    @items = []
+  end
+
+  def add_item(item)
+    @items.push(item)
```

Comment:
- [ ] I think pushing to the @items array here is unnecessary, because the `Item` class already adds to the `@items` when `add_label(self)` is called. currently what will happen is the item will be added twice into the array. maybe we should remove this `@items.push(item)` and just add an `attr_accessor` for `@items`